src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,37 @@
+.menu {
+  &__info-area {
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+  }
+
+  &__tel {
+    text-decoration: none;
+    color: #1d1d1d;
+    text-transform: uppercase;
+    font-size: 16px;
+    font-weight: 600;
+    line-height: 27px;
+    letter-spacing: 0;
+    text-align: left;
+    transition: color $animation-standard-duration ease;
```

Comment:
The text color change from #1d1d1d to black is not very noticeable and, this transition is also not showing up in the deployed site

![image](https://github.com/mate-academy/layout_landing-page/assets/52002856/5d14d389-db05-4e4c-87a4-e5aa99699190)